deployment/sandbox/flyte_generated.yaml

```diff
@@ -880,6 +881,20 @@ spec:
     app: contour
   type: NodePort
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: spark-webhook
+  namespace: sparkoperator
+spec:
+  ports:
+  - name: webhook
+    port: 443
+    targetPort: 8080
+  selector:
+    app.kubernetes.io/name: sparkoperator
+    app.kubernetes.io/version: v2.4.0-v1beta1
```

Comment:
does this need any plugin side changes?

Reply:
no, The plugin is already on v1beta1 : https://github.com/lyft/flyteplugins/blob/master/go/tasks/plugins/k8s/spark/spark.go#L19